smooth/examples/run_optimization_example.py

```diff
@@ -65,5 +66,10 @@ def main():
 if __name__ == '__main__':
     freeze_support()
     optimization_results = main()
-    # smooth.save_results('optimization_result', optimization_results)
+    for r in optimization_results:
+        print(r.smooth_result)
+
+    import smooth
```

Comment:
Write this to the other includes at the top:
```from smooth import run_optimization, save_results```
You then have to remove the ```smooth.``` in the next line.